packages/integration-tests/projects/suite-web/tests/suite/bug-report-form.test.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+
+// @group:suite
+// @retry=2
+
+describe('Stories of bug report forms', function () {
+    beforeEach(function() {
+        cy.task('startEmu', { wipe: true, version: '2.4.3' });
+        cy.task('setupEmu', { mnemonic: 'all all all all all all all all all all all all' });
+        cy.task('startBridge');
+
+        cy.viewport(1080, 1440).resetDb();
+        cy.prefixedVisit('/');
+
+    });
+
+    // todo: this test should fill report bug in Suite Guide
+    it(`Open Report form, fill bug report, submit report`, function () {
+        //goes to feedback
+        cy.getTestElement('@guide/button-open', { timeout: 40000 }).click();
```

Comment:
why such a big timeouts? can we solve this differently, this will add quite a lot of test run time.

Reply:
It only adds test run time if Suite loading takes too long, which sometimes did, and hence the big timeout. But I cut it in half so it will not eat too much time.